cli/docs/docker-cloud-migration.md

```diff
@@ -28,6 +28,7 @@ In the [`dockercloud.yml`](https://github.com/dockersamples/example-voting-app/b
 i2kit requires the following resources:
 - an access key and a secret key
 - a vpc with at least 1 subnet (we recommend 3)
+- a security group with port 22 exposed (for troubleshooting and service to service communication)
```

Comment:
can we mark this as optional? I think the pattern of deploying a single service with port 22 instead of this would be a better way. 

Do we need the SG to give access to members of the group itself?

Reply:
yeap, we need it for internal traffic. this is also the place to configure access to other AWS resources, such as RDS databases

Reply:
I can remove the port 22 example